backend/src/config/index.ts

```diff
@@ -17,17 +17,13 @@ export const getRootEncryptionKey = async () => {
 }
 export const getInviteOnlySignup = async () => (await client.getSecret("INVITE_ONLY_SIGNUP")).secretValue === "true"
 export const getSaltRounds = async () => parseInt((await client.getSecret("SALT_ROUNDS")).secretValue) || 10;
+export const getAuthSecret = async () => (await client.getSecret("AUTH_SECRET")).secretValue; // new
```

Comment:
hmm, this will mean that any existing deployments without this token will fail on next deploy. 

i think what we should do is if `JWT_AUTH_SECRET` is defined then use that as `AUTH_SECRET`. 

Option 2 is we don't have to rename `JWT_AUTH_SECRET` to `AUTH_SECRET`, this might be simpler for now and won't break existing deployments

Reply:
I'll make `AUTH_SECRET` take from `JWT_AUTH_SECRET` if it is defined; I wanna remove `JWT` from the name of the variable.

Reply:
Okay check again @maidul98